airbyte-integrations/connectors/source-salesforce/source_salesforce/spec.yaml

```diff
@@ -37,7 +37,6 @@ connectionSpecification:
       pattern: >-
         ^([0-9]{4}-[0-9]{2}-[0-9]{2}(T[0-9]{2}:[0-9]{2}:[0-9]{2}Z)?)$
       examples:
-        - "2021-07-25"
         - "2021-07-25T00:00:00Z"
```

Comment:
@a-rampalli will this impact existing connections ?

Reply:
Checked backward compatibility. No issues with syncs

Reply:
<img width="504" alt="Screenshot 2023-08-21 at 6 59 56 PM" src="https://github.com/rudderlabs/airbyte/assets/110388330/06b7b629-1b24-4bb0-b604-a759cf8ab646">
Will show an error in configuration screen for existing sources but syncs work fine. This can be fixed if user selects start date again (Will be stored in the new format in db now). Didn't add as a step for customer as it is not strictly necessary